src/content/ui/en.yaml

```diff
@@ -46,3 +46,12 @@ briefPageDescriptions:
   Libraries: Expand the possibilities of p5.js with community-created libraries.
   About: Learn about the mission, values, and people behind p5.js.
   People: Get to know the p5.js team.
+ContributePage:
+  statement: >
+    Our community values contributions of all forms and seeks to expand the meaning of the word contributor as far and wide as possible. It includes documentation, teaching, writing code, making art, writing, design, activism, organizing, curating, or anything else you might imagine.
+  imageAltText: >
+    A group of about 20 people standing and sitting together in front of a doorway with their hands up and happy looks on their faces
+  donateDescription: >
+    TEXT NEEDED HERE
+  faqDescription: >
+    TEXT NEEDED HERE
```

Comment:
@stalgiag do you think it would be better to have these in their own collection for just this page?

Reply:
I think that could make sense. I would love a more consistent key naming strategy. Currently I believe the keys that lead to a value are pascal case and the keys that hold arrays are camel case.

Reply:
i'll standardize the naming for now and we can revisit if these should be in a different place once we get through a few more pages